packages/postcss-merge-rules/src/index.js

```diff
@@ -13,15 +13,17 @@ function intersect (a, b, not) {
     });
 }
 
-const checkMS = (a, b) => !!(~a.indexOf('ms') && ~b.indexOf('ms'));
+
+// Internet Explorer use :-ms-input-placeholder.
+// Microsoft Edge use ::-ms-input-placeholder.
+const findMsInputPlaceholder = selector => ~selector.indexOf('-ms-input-placeholder');
```

Comment:
Better use `regex` with `i` flag, because css is case insensitivity language

Reply:
Oh, well, I did not think about it